contractcourt/channel_arbitrator.go

```diff
@@ -3160,6 +3049,192 @@ func (c *ChannelArbitrator) checkLegacyBreach() (ArbitratorState, error) {
 	return StateContractClosed, nil
 }
 
+// sweepRequest wraps the arguments used when calling `SweepInput`.
+type sweepRequest struct {
+	// input is the input to be swept.
+	input input.Input
+
+	// params holds the sweeping parameters.
+	params sweep.Params
+}
+
+// createSweepRequest creates an anchor sweeping request for a particular
+// version (local/remote/remote dangling) of the commitment.
```

Comment:
another terminology nitpick. What is a remote dangling *anchor*? Are we referring to an anchor sweep that includes remote dangling HTLCs? Are we referring to an anchor sweep on the *pending* remote commitment? I think this PR could use a comprehensive review of the terminology used and make sure that we are *very consistent* with it since there are so many delicate cases that need to be handled correctly.

Reply:
If you have a good alternative I will reconsider the naming here, but could you mention the part where we actually use the wording `remote dangling anchor`

Reply:
you are correct, will replace with pending instead of dangling.